client/package.json

```diff
@@ -1,17 +1,56 @@
 {
-  "name": "trypurescript-client",
-  "private": true,
+  "name": "tps",
```

Comment:
Renamed from `trypurescript-client` to make changing configs with these commands a bit less verbose:
```
npm config set tps:configpath "config/dev/*.purs"
npm config set tps:configpath "config/prod/*.purs"
```